.github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -5,17 +5,24 @@ title: '[PROPOSAL]'
 labels: proposal
 assignees: ''
 ---
-**What kind of business use case are you trying to solve? What are your requirements?**
-A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
 
-**What is the problem? What is preventing you from meeting the requirements?**
-Describe what impact this problem has for lack of a solution.
+### What are you proposing?
+_In a few sentences, describe the feature and its core capabilities._
 
-**What are you proposing? What do you suggest we do to solve the problem or improve the existing situation?**
-Describe your proposed solution. It's OK if you don't have one.
+### How did you come up with this proposal?
+_Highlight any research, proposals, requests, issues, forum posts, anecdotes that signal this is the right thing to build. Highlight opportunities for additional research._
 
-**What are your assumptions or prerequisites?**
-Describe any assumptions you may be making that would limit the scope of this proposal.
+### What is the user experience going to be?
+_Describe the feature requirements and or user stories. You may include low-fidelity sketches, wireframes, APIs stubs, or other examples of how a user would use the feature. Using a bulleted list or simple diagrams to outline features is okay. e.g. As a < type of user > , I want to < achieve a goal > so that < for some reason >._
 
-**What are remaining open questions?**
-List questions that may need to be answered before proceeding with an implementation.
+### Why should we build this?
+_Describe the most important user needs, pain points, and the value that this feature will bring to the OpenSearch community, as well as what impact it has if it isn't built. What is preventing you from meeting this need today?_
+
+### Why shouldn't we build this? (Optional, but encouraged)
```

Comment:
A couple of smaller items here. I think we shouldn't say "we" and merge these two. How about this?

"Why should it be built? Any reason not to?"
 _Describe the most important user needs, pain points, and the value that this feature will bring to the OpenSearch community, as well as what impact it has if it isn't built, or new risks if it is. What is preventing you from meeting this need today?_

Reply:
I'm good with this wording.

Reply:
@dblock i've updated using the language you recommended. If it looks good to you, please approve and merge.